packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -18,14 +18,32 @@ const addFileSharingSnippet = `
 <CallWithChatComposite
 adapter={chatAdapter}
 options={{
-  fileSharing: {
-    uploadHandler: attachmentUploadHandler,
-    /* If fileDownloadHandler is not provided. The file URL is opened in a new tab.
-    You can find examples of fileDownloadHandler and attachmentUploadHandler in this tutorial
+  attachmentOptions: {
+    uploadOptions: {
+      supportedMediaTypes: ['image/png', 'image/jpeg', 'text/plain', '.docx'],
```

Comment:
Should `.docx` be here? Looks like a file extension rather than a media type?

Reply:
good catch, i will blame copilot...